inspec.yml

```diff
@@ -33,42 +32,24 @@ attributes:
     type: numeric
```

Comment:
I think all these type names need to be the formal ruby object names ... aka `Array` vs `array` and `Numberic` vs `numeric`

Reply:
Done